### DISCUSSION TITLE
The Budibase client library could not be loaded

### DISCUSSION THREAD

**Discussion by leuedaniel** [Budibase/budibase#12696] [Help]
**Checklist**
- [X] I have searched budibase discussions and github issues to check if my issue already exists

**Hosting**

- Self
  - Method: Docker single Image 
  - Budibase Version: 2.13.50 

**Describe the bug**
When I publish an app, I can easily open it with the link. As soon as I open the app in another browser, the message appears:
There was an error loading your app
The Budibase client library could not be loaded. Try republishing your app.

Generally, when I close the browser and open the link again, it sometimes works and sometimes not.

**To Reproduce**
It is difficult to reproduce as it does not always occur.

**Screenshots**
![image](https://github.com/Budibase/budibase/assets/44936305/d4b634dc-c681-420f-b519-e5334a3b16f9)

**Logfile**
`{"level":"ERROR","timestamp":"2023-12-27T08:12:55.072Z","service":"@budibase/server","pid":281,"tenantId":"default","correlationId":"ef4e46d7-cb57-42b0-a656-e91cf8169b02","msg":"Auth Error: Session not found - us_f567de77f372402989f4e608bdc2cc8e - d985e2ecfa2342568e9eb515085be6fb"}
{"level":"ERROR","timestamp":"2023-12-27T08:12:55.072Z","service":"@budibase/server","err":{"type":"Error","message":"Session not found - us_f567de77f372402989f4e608bdc2cc8e - d985e2ecfa2342568e9eb515085be6fb","stack":"Error: Session not found - us_f567de77f372402989f4e608bdc2cc8e - d985e2ecfa2342568e9eb515085be6fb\n    at getSession (/app/dist/index.js:109282:11)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async /app/dist/index.js:151394:21\n    at async /node_modules/koa-compress/lib/index.js:38:5\n    at async errorHandling (/app/dist/index.js:155897:5)\n    at async /node_modules/koa-mount/index.js:52:26\n    at async userAgent (/node_modules/koa-useragent/dist/index.js:12:5)"},"pid":281,"tenantId":"default","correlationId":"ef4e46d7-cb57-42b0-a656-e91cf8169b02","msg":""}
`

**Comment by linear** [comment]
<p><a href="https://linear.app/budibase/issue/BUDI-7860/the-budibase-client-library-could-not-be-loaded">BUDI-7860 The Budibase client library could not be loaded</a></p>
